src/openrct2/localisation/Formatting.cpp

```diff
@@ -587,7 +587,16 @@ namespace OpenRCT2
                 {
                     auto month = DateGetMonth(arg);
                     auto year = DateGetYear(arg) + 1;
-                    FormatMonthYear(ss, month, year);
+                    FormatMonthYear(ss, month, year, false);
+                }
+                break;
+            case FormatToken::MonthYearSentence:
+                if constexpr (std::is_integral<T>())
+                {
+                    auto month = DateGetMonth(arg);
+                    auto year = DateGetYear(arg) + 1;
+                    FormatMonthYear(ss, month, year, true);
+                    ss << '.';
```

Comment:
Is this period necessary? I would say it's up to the translator to include it where necessary.
```suggestion
```